asteroid/separate.py

```diff
@@ -88,6 +90,11 @@ def separate(
 @torch.no_grad()
 def torch_separate(model: Separatable, wav: torch.Tensor, **kwargs) -> torch.Tensor:
     """Core logic of `separate`."""
+    if model.n_channels is not None and wav.shape[-2] != model.n_channels:
+        raise RuntimeError(
+            f"Model supports {model.n_channels}-channel inputs but found audio with {wav.shape[-2]} channels."
+            f"Please match the number of channels."
+        )
```

Comment:
Should we add a flag to ignore that, if passed, make us ignore that and take the first channels or something? 
Something that would be passed from the CLI to here (`--ignore-channels-check`).
I'm not sure it's useful.

Reply:
Not sure...

Reply:
@popcornell what's your opinion on that? 
Maybe we can start by not having it. And if we find it useful later, or there is a user demand, we can change that.